src/lib/comms/sol-network-impl-zephyr.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <errno.h>
+
+#include <net/net_core.h>
+#include <contiki/ip/uiplib.h>
+
+#include "sol-log.h"
+#include "sol-network.h"
+#include "sol-util.h"
+#include "sol-vector.h"
+
+static struct sol_vector links = SOL_VECTOR_INIT(struct sol_network_link);
+
+static inline char
+to_hex(uint8_t num)
```

Comment:
On Mon, 28 Mar 2016 11:32:29 -0700, Vinicius Gomes wrote:
> > + */
> > +
> > +#include <errno.h>
> > +
> > +#include <net/net_core.h>
> > +#include <contiki/ip/uiplib.h>
> > +
> > +#include "sol-log.h"
> > +#include "sol-network.h"
> > +#include "sol-util.h"
> > +#include "sol-vector.h"
> > +
> > +static struct sol_vector links = SOL_VECTOR_INIT(struct sol_network_link);
> > +
> > +static inline char
> > +to_hex(uint8_t num)
> 
> Unused function?

Interesting, I can vividly remember deleting it.